python/k4FWCore/ApplicationMgr.py

```diff
@@ -39,6 +41,12 @@ class ApplicationMgr:
 
     def __init__(self, **kwargs):
         self._mgr = AppMgr(**kwargs)
+        try:
+            self._mgr.OutputLevel = getattr(Configuration, logging.getLevelName(logger.level))
+        except AttributeError:
```

Comment:
This except should never be reached if using `set_log_level` (under normal running). I guess we can leave it here in case someone gets the logger and changes the level manually.

Reply:
I would prefer to keep it, because it's not required for users to go through `set_log_level` and this at least gives them a warning that something didn't work.

Should we fall back to `INFO` in that case? Or is that the default already?

Reply:
If using k4run then `INFO` is the default, if not I'm not sure.